project_stock/models/account_analytic_line.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    task_id = fields.Many2one(comodel_name="project.task", string="Project Task",)
+
+    def _timesheet_postprocess_values(self, values):
+        res = super()._timesheet_postprocess_values(values)
+        # Delete the changes in amount if the analytic line come from task.
```

Comment:
Why is this needed?

Reply:
I have verified with the tests that this is not necessary.

Reply:
Comment added in the function to explain the reason.